### PR TITLE
ADD-FOREIGN-KEY

### PR DESCRIPTION
**Pull request by muneeb3123**
THIS PR INCLUDES:
Create a table named owners with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
full_name: string
age: integer
Create a table named species with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
name: string
Modify animals table:
Make sure that id is set as autoincremented PRIMARY KEY
Remove column species
Add column species_id which is a foreign key referencing species table
Add column owner_id which is a foreign key referencing the owners table
Insert the following data into the owners table:
Sam Smith 34 years old.
Jennifer Orwell 19 years old.
Bob 45 years old.
Melody Pond 77 years old.
Dean Winchester 14 years old.
Jodie Whittaker 38 years old.
Insert the following data into the species table:
Pokemon
Digimon
Modify your inserted animals so it includes the species_id value:
If the name ends in "mon" it will be Digimon
All other animals are Pokemon
Modify your inserted animals to include owner information (owner_id):
Sam Smith owns Agumon.
Jennifer Orwell owns Gabumon and Pikachu.
Bob owns Devimon and Plantmon.
Melody Pond owns Charmander, Squirtle, and Blossom.
Dean Winchester owns Angemon and Boarmon.

- Write queries (using JOIN) to answer the following questions:

1. What animals belong to Melody Pond?
Melody_animals

3. List of all animals that are pokemon (their type is Pokemon).
type_pokemon

5. List all owners and their animals, remember to include those that don't own any animal.
all_owners

7.How many animals are there per species?
animals_per_species

8.List all Digimon owned by Jennifer Orwell.
Screenshot from 2023-06-15 20-53-23

9.List all animals owned by Dean Winchester that haven't tried to escape.
Dean_escape_animals

10.Who owns the most animals?
most_animals

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -7,6 +7,25 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL(5, 2),
-    species VARCHAR(50),
+    species_id INT,
+    owner_id INT,
+    FOREIGN KEY(owner_id) REFERENCES owners(id) ON DELETE CASCADE,
+    FOREIGN KEY(species_id) REFERENCES species(id) ON DELETE CASCADEs,
     PRIMARY KEY(id)
```
Comment (uwadonat) [inline schema.sql]: - In this project, you are supposed to modify the animal's table as below and add commands in this `schema.sql` file.:

- [ ] Remove column species
- [ ] Add column species_id which is a foreign key referencing species table
- [ ] Add column owner_id which is a foreign key referencing the owners table

`Hint`: Think about `ALTER` command

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @muneeb3123,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- Well-organized and clean project-  ✔️
- Detailed pull request message ✔️
almost

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (muneeb3123) [reply schema.sql]: ok sure i understand

**Review by Munsa1 — APPROVED** [review]
Hi @muneeb3123, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Good Commit Message :heavy_check_mark:  

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._ :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**